GoogleSignIn/Sources/GIDToken.m

```diff
@@ -55,4 +55,37 @@ - (void)encodeWithCoder:(NSCoder *)encoder {
   [encoder encodeObject:_expirationDate forKey:kExpirationDateKey];
 }
 
+#pragma mark - isEqual
+
+- (BOOL)isEqual:(id)object {
+  if (self == object) {
+    return YES;
+  }
+  if (![object isKindOfClass:[GIDToken class]]) {
+    return NO;
+  }
+  return [self isEqualToGIDToken:(GIDToken *)object];
+}
+
+- (BOOL)isEqualToGIDToken:(GIDToken *)other {
```

Comment:
I don't think you need to have the full type as the last phrase of this method. Consider `-[NSDate isEqualToDate:]` as opposed to `-[NSDate isEqualToNSDate:]`.

Reply:
Definitely don't need the full type.